pandas/tests/groupby/test_function.py

```diff
@@ -555,6 +555,83 @@ def test_idxmin_idxmax_axis1():
         gb2.idxmax(axis=1)
 
 
+@pytest.mark.parametrize("numeric_only", [True, False, None])
+def test_axis1_numeric_only(groupby_func, numeric_only):
+    if groupby_func in ("idxmax", "idxmin"):
+        pytest.skip("idxmax and idx_min tested in test_idxmin_idxmax_axis1")
+    if groupby_func in ("mad", "tshift"):
+        pytest.skip("mad and tshift are deprecated")
+    if groupby_func in ("corrwith", "skew"):
+        msg = "GH#47723 groupby.corrwith and skew do not correctly implement axis=1"
```

Comment:
Could you xfail this case? Or if these cases work would the resulting test not work/be applicable?

Reply:
Good call; done.